MAINTAINERS.md

```diff
@@ -1,21 +1,25 @@
 # Maintainers
 
-MAINTAINERS:
+Code for this project is reviewed by any other member of the team before it is merged into the project. Team members include:
 
-- lucasmbrown-usds
-- aprilypchen
+## Project Leadership
 
-approvers:
+* Lucas Brown
```

Comment:
Should we add Julius too?

Reply:
He isn't maintaining the project from a code or design perspective, creating assets, or working in the repository, so my inclination is to leave him off. We can discuss.